freshmaker/handlers/koji/rebuild_images_on_odcs_compose_done.py

```diff
@@ -53,6 +53,10 @@ def handle(self, event):
             ArtifactBuild.state == ArtifactBuildState.PLANNED.value,
             Compose.odcs_compose_id == event.compose['id'],
             ArtifactBuildCompose.compose_id == Compose.id)
+
+        if builds_ready_to_rebuild:
+            self.log_info('ODCS compose %s finished', event.compose['id'])
```

Comment:
Shouldn't this logging be without `if`? So we see a message about done compose every time handler is triggered and not only when there are builds waiting for this compose?

Reply:
We only cares about composes that requested by Freshmaker (there are builds in db connect to this compose when it was requested by Freshmaker), composes requested by other clients (like osbs) can simply be ignored.

Reply:
Thanks for clarification